overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -222,7 +222,7 @@ func (s *servicectlSuite) SetUpTest(c *C) {
 		snapstate.EnforcedValidationSets = old
 	})
 	snapstate.EnforcedValidationSets = func(st *state.State, extraVss ...*asserts.ValidationSet) (*snapasserts.ValidationSets, error) {
-		return nil, nil
+		return snapasserts.NewValidationSets(), nil
```

Comment:
what's the reason for the change?

Reply:
In the previous version of the code that checked the validation sets against the to-be-installed snaps, we had `nil` checks in places where we had already confirmed that the error returned from `snapstate.EnforcedValidationSets` was `nil`. Since the actual implementation of `snapstate.EnforcedValidationSets` never returns `nil, nil`, I was able to simplify the calling code to eliminate those redundant `nil` checks, but I had to update the mocked implementations.